Neuro/Tasks/TasksHandler.cs

```diff
@@ -174,28 +174,31 @@ private void Update()
         if (Minigame.Instance) return;
         if (!PlayerControl.LocalPlayer) return;
 
-        foreach (PlayerTask task in PlayerControl.LocalPlayer.myTasks)
+        //Open anything nearby, speedrun.
+        foreach (var tasky in PlayerControl.LocalPlayer.myTasks)
         {
-            if (task.IsComplete || task.Locations == null) continue;
+            if (tasky.IsComplete || tasky.Locations == null) continue;
 
-            // TODO: HOLY FUCK DON'T UNCOMMENT THIS THIS IS BROKEN AF
-            // INVOKE CONSOLE.USE DIRECTLY PLEEEEEEASE
+            Il2CppSystem.Collections.Generic.List<Console> list = tasky.FindConsoles();
```

Comment:
I don't see the point of not inlining this in the foreach loop

Reply:
> I don't see the point of not inlining this in the foreach loop

It's consoles spesific to that task, so it needs to be there.

Reply:
Sorry i misunderstood.